ramler-itest/src/test/java/org/ops4j/ramler/itest/FluentSetterTest.java

```diff
@@ -0,0 +1,35 @@
+package org.ops4j.ramler.itest;
+
+import org.junit.Test;
+import org.ops4j.ramler.itest.model.Employee;
+import org.ops4j.ramler.itest.model.Person;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Tests that fluent setters are working
```

Comment:
General remark: It is good practice to terminate each sentence in Javadoc with a full stop. (".")

This is not just nitpicking, since the javadoc tool uses it to separate summary from details.